assets/src/dashboard/app/views/myStories/header/index.js

```diff
@@ -15,15 +15,16 @@
  */
 
 /**
- * External dependencies
+ * WordPress dependencies
  */
```

Comment:
I asked @swissspidy and fixed the order in a few places because it felt random. Was this added by ESLint?

> It should be External, WordPress, Internal, but the ESLint rule is unfortunately not very smart and sometimes the order gets mixed up

Reply:
Ooops I'll revert this, I was used to seeing WP -> External -> Internal

Reply:
this is news to me! i thought it was WP -> External -> Internal. I'll do a better job monitoring this in the dashboard.